contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -35,8 +36,8 @@ abstract contract InitializableAbstractStrategy is Initializable, Governable {
     address[] internal assetsMapped;
 
     // Reward token address
-    address public rewardTokenAddress;
-    uint256 public rewardLiquidationThreshold;
+    address[] public rewardTokenAddresses;
```

Comment:
We need to depreciate these two slots and add the new ones after. (and decrement the reserved appropriately.

Reply:
I was thinking that since arrays always take a new & full solidity storage slot we might get away with just re-setting both of the array values for all of the strategies. (Haven't confirmed this with on-chain testing yet though - will do that today or tomorrow). 

I you think the deprecation and decrementing is a better approach I can still do that.

Reply:
I still would rather depreceate.